packages/ui-informed/src/controls/formControl/FormMessage.tsx

```diff
@@ -0,0 +1,16 @@
+import { cn } from '@utima/ui';
+import { forwardRef, type ComponentProps } from 'react';
+
+export type FormMessageProps = ComponentProps<'p'>;
+
+export const FormMessage = forwardRef<HTMLDivElement, FormMessageProps>(
```

Comment:
replace forward ref with `memo`, it's not needed in this case (done incorrectly in wue)